pandas/tseries/period.py

```diff
@@ -376,6 +389,11 @@ def _to_embed(self, keep_tz=False):
     def _formatter_func(self):
         return lambda x: "'%s'" % x
 
+    @property
+    def _int64index(self):
+        # do not cache, same as .asi8
```

Comment:
Followed the comment on ``numeric.py``. I can't find original issue for memory leak.

- https://github.com/pydata/pandas/blob/master/pandas/indexes/numeric.py#L135